### PR TITLE
Resolved some issues from #issue15

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### PR Description 
1. Added support to update the title. 
2. Now not adding Issues if those are unassigned 
3. Adding all issues all assigned (even if it is multi-assigned) 
4. help to resolve some issue from #15

### CODE REVIEW

File: jirasync.py [hunk 1]
```diff
@@ -314,7 +315,6 @@ def start_syncing_team(config, interval):
                 teams[team]['github_token']
             )
 
-            created_issue_list = git_hub_plugin.get_github_issues_created_list()
```
Comment (omkarkhatavkar) [inline jirasync.py:317]: @rochacbruno Here was the code trying to fetch issue list even it was unassigned.

File: plugins/github.py [hunk 2]
```diff
@@ -64,9 +64,9 @@ def get_github_issues_assigned_list(self, status='None'):
         stats = []
         for issue_stat in all_stats[:]:
             assignees = len(issue_stat.data['assignees'])
-            if assignees == 1 and status is None:
+            if assignees > 1 and status is None:
                 stats.append(issue_stat)
-            elif assignees == 1 and status == str(issue_stat.data['state'].encode()):
+            elif assignees > 1 and status == str(issue_stat.data['state'].encode()):
```
Comment (omkarkhatavkar) [inline plugins/github.py:69]: The increasing flow here and assigned issues will now sync to Jira

File: wrappers/workflow_wrapper.py [hunk 3]
```diff
@@ -67,12 +67,19 @@ def create_pull_request_review_in_current_sprint(issue, jira, issue_text, assign
...
+def update_existing_issue_in_current_sprint(task, issue, jira, issue_text):
     if isinstance(jira, MyJiraWrapper):
         if task.fields.status.name.encode('utf-8') == 'Done':
             echo_skip("No Need of Update! Already Done!")
         else:
-            if issue.data['state'].encode('utf-8') == 'closed':
+            # change the title id it was update in source (e.g. github)
+            if issue.title.encode('utf-8') not in task.fields.summary.encode('utf-8'):
+                import pdb
+                pdb.set_trace()
```
Comment (rochacbruno) [inline wrappers/workflow_wrapper.py]: There is a missing `pdb` here

Comment (omkarkhatavkar) [reply wrappers/workflow_wrapper.py]: thanks for identifying let me remove that

**Pull request merged**